ui/app/templates/components/calendar-widget.hbs

```diff
@@ -107,15 +103,14 @@
         </div>
         <div class="calendar-widget-grid calendar-widget">
           {{#each this.widgetMonths as |month|}}
-            <button
-              data-test-calendar-month={{month.name}}
-              type="button"
+            <Hds::Button
+              @text={{month.name}}
+              @color="secondary"
               class="is-month-list {{if month.readonly 'is-readOnly'}}"
```

Comment:
If you think it's worth it - maybe as a follow-on we can clean up some of these classes so we don't have too many one-off button styles.

Reply:
I figured for this one it would require design input and we should handle that separately from this PR

Reply:
Good call!